### ISSUE TITLE
Unable to use AWS4-HMAC-SHA256

### ISSUE THREAD

**Issue by rmaspero** [dwilkie/carrierwave_direct#217]
I believe I've followed all the instructions but seem to get the following error

```
<Error>
<Code>InvalidRequest</Code>
<Message>
The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256.
</Message>
<RequestId>77007434876C64A9</RequestId>
<HostId>
CABOhE9/c+mAMk75qDnm8/d/T2rjl4BLI4Qndre/HUZWJLjhjPitCvDqN79MOTiAHUI76V+b8Iw=
</HostId>
</Error>
```

Carrierwave works normally, but when I try and do Carrierwave_direct I get this message. My S3 bucket is in London, I don't know if this might affect it. I've been over everything again and again but can't seem to solve this?

**Comment by rmaspero** [comment]
I was able to solve the issue by changing my buckets location to an older location which supports the older signatures. Not really a fix, but now works.

**Comment by alkami-io** [comment]
Is there anymore info on this particular issue?  I'm having the issue with a bucket on us-east-2 and have not come across any info on how to resolve it.  

Thanks.

**Comment by runephilosof** [comment]
You have to use the master branch since the fix for this has not been released yet.

**Comment by p8** [comment]
I've just released version 1.0.0.
